docs/overview.md

```diff
@@ -22,9 +22,7 @@ The program that creates the inputs is called a fuzzer.
 Fuzzing is highly effective at finding bugs missed by manually written tests,
 code review, or auditing.
 Fuzzing has found thousands of bugs in mature software such as Chrome, OpenSSL,
-and Curl.
-If fuzzing isn't finding bugs in your code, your code is probably not bug free,
-you are probably not fuzzing well.
+and Curl. When done well, fuzzing is able to find bugs in virtually any code.
```

Comment:
One thing I try to do when writing markdown is to start every sentence on a new line.
This makes merge conflicts less likely.
It's not necessary but I think it's a good idea and can make things a little more readable IMO.

Reply:
Sure, thanks for the tip! Will do going forward.

Reply:
Thanks!